pkgs/tools/filesystems/ceph/default.nix

```diff
@@ -136,6 +137,7 @@ in rec {
       # for pybind/rgw to find internal dep
       export LD_LIBRARY_PATH="$PWD/build/lib:$LD_LIBRARY_PATH"
       # install target needs to be in PYTHONPATH for "*.pth support" check to succeed
+      export PYTHONPATH=${ceph-python-env}/lib/python${pyVersion}/site-packages:$lib/lib/python${pyVersion}/site-packages/:$out/lib/python${pyVersion}/site-packages/
```

Comment:
can we add a line here
```
# set PYTHONPATH, so the build system doesn't silently skip installing `ceph-volume`
```
?

Reply:
@srhb @flokli Why was this needed?

Isn't this what

```
pythonPath = [ ceph-python-env "${placeholder "out"}/${ceph-python-env.sitePackages}" ];
```

does? (I haven't found yet where exactly the usage of `pythonPath` in `mkDerivation` is implemented)

I found it confusing that we currently set both, so it would be nice to understand _why_ `pythonPath` isn't enough.

Reply:
I'm not sure. @srhb ?